extension/manifest.json

```diff
@@ -15,7 +15,8 @@
       "matches": [
         "https://*.github.com/*",
         "https://github.com/*",
-        "https://git.musta.ch/*"
+        "https://git.musta.ch/*",
+        "https://git.autodesk.com/*"
```

Comment:
@shaneosullivan I don't think we want private URLs in here. We probably just want to make this configurable for people. https://github.com/sindresorhus/refined-github does this, we can crib that from them.